apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/main/java/gov/cms/model/dsl/codegen/plugin/model/ExternalTransformationBean.java

```diff
@@ -0,0 +1,24 @@
+package gov.cms.model.dsl.codegen.plugin.model;
+
+import gov.cms.model.dsl.codegen.library.ExternalTransformation;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Model object for defining the name of an external transformation function to call when
+ * transforming a message object into an entity.
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class ExternalTransformationBean {
+  /**
+   * Name of the constructor parameter and field in the generated transformation class by which this
+   * external {@link ExternalTransformation} will be referenced. A mapping can have zero or more of
+   * these.
+   */
+  private String name;
+}
```

Comment:
Is there a specific need for a one property class?  Could this not just be referenced as a string by whatever consuming logic?

Reply:
Yes this makes the types explicit in the model and also leaves room to expand the properties of external transformations later without having to do any big refactoring.